NOTICE

```diff
@@ -0,0 +1,5 @@
+Student Explorer
+Copyright © 2015 The Regents of the University of Michigan
```

Comment:
A grep returns copyright years as "2015 - 2017". 

0587374590:student_explorer zqian$ find . -name "*.*" | xargs grep -i -B 1 -A 1 "Regent"
grep: .: Is a directory
grep: ./.git: Is a directory
grep: ./.idea: Is a directory
grep: ./seumich/static/seumich/images/Status_Icons_Not: No such file or directory
grep: Applicable.png: No such file or directory
./student_explorer/templates/student_explorer/index.html-                        <span class="footer-block">Copyright &copy; 2015 - 2017</span>
./student_explorer/templates/student_explorer/index.html:                        <span class="footer-block">The Regents of the University of Michigan</span>
./student_explorer/templates/student_explorer/index.html-                    </p>

Please update the year to be "2015 - 2017".

Reply:
No.  Tech Transfer instructed that the copyright declaration list the year in which the project started only, no range.

> From: Drew Bennett <andbenne@umich.edu>
> Subject: Re: Open Source Software Licensing for Student Explorer
> Date: November 7, 2017 at 7:12:49 AM PST
> To: Anthony Whyte <arwhyte@umich.edu>
> Cc: Daniel Burgess <burgessd@umich.edu>

> Hi Anth,

> We are fine with either BSD or Apache, MIT while acceptable has a number of gaps that are addressed in a more fully vetted approach such as BSD or Apache. Apache is fine from our point of view, copyright attribution notice should be UM standard “(c) 2017 The Regents of the University of Michigan”.

> The year should be adjusted to the year that the first version was built, it should NOT be a range. You should also feel free to add an additional description of authors, labs involved, etc. below the copyright notice if that is something that the team would like to do.